### PR TITLE
add unique processId creation in PolicyController, adapt+amend tests

### PR DESCRIPTION
**Pull request by matthiaszimmermann**

### CODE REVIEW

**Comment by matthiaszimmermann** [comment]
review still pending

File: contracts/modules/PolicyController.sol [hunk 1]
```diff
@@ -31,18 +31,19 @@ contract PolicyController is
     mapping(bytes32 => mapping(uint256 => Payout)) public payouts;
     mapping(bytes32 => uint256) public payoutCount;
 
-    bytes32[] private _processIds;
+    uint256 private _processIds;
```
Comment (doerfli) [inline contracts/modules/PolicyController.sol:34]: This name should be changed to `_processId` or `_lastProcessId`.

File: contracts/modules/PolicyController.sol [hunk 2]
```diff
@@ -442,6 +442,18 @@ contract PolicyController is
     }
 
     function processIds() external view returns (uint256) {
```
Comment (doerfli) [inline contracts/modules/PolicyController.sol:444]: Should be changed accordingly

File: contracts/modules/PolicyController.sol (hunk 1 above)
Comment (matthiaszimmermann) [reply contracts/modules/PolicyController.sol:34]: renamed member to "_assignedProcessIds", committed on master

**Pull request merged**